src/Endpoints/DataTransferObjects/Organisation/OrganisationRelationships.php

```diff
@@ -13,5 +13,5 @@
  */
 class OrganisationRelationships extends DataTransferObject
 {
-    public RelationshipRelatedMany $associations;
+    public ?RelationshipRelatedMany $associations;
```

Comment:
Based on our tests it is not possible for the association relationship to be empty. Could you please provide the exception stack trace you are receiving and the date and time (to the second) when the api call was made, so we can inspect this further?

Reply:
For some reason I was not able to reproduce this now, but I have the logs from this morning.
Happened at 04:35 and 04:36 GMT, while trying to get organisation details.
I had the same issue back in August when I first tried using the API, so it was not an isolated incident of this morning.

> An Error Occurred loading widget pingen: Invalid type: expected `Pingen\Endpoints\DataTransferObjects\Organisation\OrganisationRelationships::letters` to be of type `Pingen\Endpoints\DataTransferObjects\General\RelationshipRelatedMany`, instead got value `null`, which is NULL..